test/flops.h

```diff
@@ -107,8 +107,7 @@ static double fmuls_gbmm(double m, double n, double k, double ku, double kl)
 {
     // recall A is an m x k matrix, and it is the only band matrix
     double elements, k_leftover;
-    // elements from kl
-    elements += (ku+1+kl)*(k<m?k:m);
+    elements = (ku+1+kl)*(k<m?k:m);
```

Comment:
Not sure where this pile-up of PRs comes from. LAWN41 on page 118 gives a very simple formula for this: www.netlib.org/lapack/lawnspdf/lawn41.pdf without any conditionals or min/max that should have been used instead of the ternary `?:` operator.